src/styles/embed-reset.css

```
@@ -1,12 +1,17 @@
-  all: initial;
+  /* Create an isolated styling context without nuking inherited tokens */
+  display: block;
+  isolation: isolate;
+  width: 100%;
+  height: 100%;
+  background: inherit;
+  color: inherit;
   box-sizing: border-box;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   line-height: 1.5;
   color: #333;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)